docs/application/native/guides/alarm/alarms.md

```diff
@@ -182,10 +186,17 @@ on_foreach_registered_alarm(int alarm_id, void *user_data)
             dlog_print(DLOG_INFO, TAG, "Alarm Recurrence on SATURDAY \n");
     }
 
-    /* Cancel scheduled alarms */
-    ret = alarm_cancel(alarm_id);
-    if (ret != ALARM_ERROR_NONE)
-        dlog_print(DLOG_ERROR, TAG, "Cancel Error: %d ", ret);
+	if (alarm_id == to_update_alarm_id) {
```

Comment:
Kindly check the indentation for the updated code to match with existing one and left align. Please check the screenshot attached regarding the same:
![image](https://user-images.githubusercontent.com/42992085/61711911-c9800c80-ad72-11e9-9530-13d64482db5a.png)

Reply:
Done